tests/integration/test_plugins.py

```diff
@@ -287,5 +287,4 @@ async def test_thinbackup_plugin(ops_test: OpsTest, unit_web_client: UnitWebClie
         "ssh", "--container", "jenkins", unit_web_client.unit.name, "ls", backup_path
     )
     assert ret == 0, f"Failed to ls backup path, {stderr}"
-    # The backup folder is of format FULL-<backup-date>
-    assert "FULL" in stdout
+    assert "FULL" in stdout  # The backup folder is of format FULL-<backup-date>
```

Comment:
Sorry, I  meant `assert "FULL" in stdout, The backup folder is not of format FULL-<backup-date>`

Reply:
Just to make sure, you mean like this?

```
# assert "FULL" in stdout, the backup folder is of format FULL-<backup-date>
assert "FULL" in stdout
```

Reply:
No, I mean removing the comment and instead adding the explanation to the `assert` statement

Reply:
Done!